observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dist-tracing-tempo-troubleshooting"]
+= Troubleshooting
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-tempo-troubleshooting
+
+toc::[]
+
+You can diagnose and fix issues in TempoStack or TempoMonolithic instances by using various troubleshooting methods.
```

Comment:
Should Tempo and TempoMonolithic be in backticks?

Reply:
Thank you. Here, it is used to mean a product name rather than a CR name. It is used as such consistently without backticks elsewhere in this doc set. I will revisit and review this for another Jira.